docs/validators/redundancy.md

```diff
@@ -0,0 +1,48 @@
+---
+id: redundancy
+title: Node redundancy
+---
+
+Each validator node can be backed up by one or several backup nodes in order to 
+provide security of the network and to assure that a node will unlikely get jailed.
+
+The redundancy mechanism relies on the following principle: the owner of the BLS key will start `n` nodes
+with almost the same configuration, and the same `validatorKey.pem` file. The difference between 
+configurations consists on an option inside the `prefs.toml` file.
```

Comment:
Replace:
The redundancy mechanism relies on the following principle: the owner of the BLS key will start `n` nodes
with almost the same configuration, and the same `validatorKey.pem` file. The difference between 
configurations consists on an option inside the `prefs.toml` file.

with:
This is a redundancy mechanism which allows the Main Validator operator to start additional 'n' hot-standby nodes, each of them running the same 'validatorKey.pem' file. 

Hot standby nodes are configured using the 'RedundancyLevel' option in the 'prefs.toml' configuration file.

Reply:
done